src/components/upload/FileUpload.js

```diff
@@ -40,40 +48,72 @@ const FileUpload = () => {
     const [state, setState] = useState("idle");
     const [projectFile, setProjectFile] = useState();
     const [projectName, setProjectName] = useState("");
+    const [open,setOpen] = React.useState(false);
+    const handleClose = () => {
+        setOpen(false);
+      };
 
-    const onSubmit = () => {
+      const onSubmit = () => {
+        setOpen(true);
+        uploadProjectFile(projectName, projectFile)
+      };
+    /*const onSubmit = () => {
         setState("uploading");
         uploadProjectFile(projectName, projectFile)
             .then(() => {
                 Notifications.notify("Project created.", "success")
                 setState("idle");
+                setOpen(true);
+                return (
+                    <Dialog
+                    open={open}
+                    onClose={handleClose}>
+                     <DialogTitle> Edit projects </DialogTitle>
+                        <DialogContent>
+                            <DialogContentText>
+                                 <EditorPage></EditorPage>
+                            </DialogContentText>
+                        </DialogContent>
+               </Dialog>
+                );
             })
             .catch((err) => {
                 Notifications.notify(`Failed to create project.\n${err}`, "error")
                 setState("idle");
             })
-    };
+    };*/
 
     return (
         <FileUploadRoot>
             <FileDropZone
                 file={projectFile}
                 onFile={setProjectFile}
             />
-
+                
             <Confirmation>
-                <Input
+               <Box  component="form"  sx={{ '& > :not(style)': { m: 1, width: '25ch' },  }}
+                     noValidate autoComplete="off" >
```

Comment:
```suggestion
               <Box>
```

1. We don't need a form component here
2. The style shouldn't be applied inline - i.e. no `sx` property but a styled component.
3. Remove the non-effective properties